src/components/application_manager/include/application_manager/commands/hmi/response_from_hmi.h

```diff
@@ -61,6 +61,14 @@ class ResponseFromHMI : public CommandImpl {
   void CreateHMIRequest(const hmi_apis::FunctionID::eType& function_id,
                         const smart_objects::SmartObject& msg_params) const;
 
+  /**
+   * @brief Extracts message param value from smart object
+   *
+   * @param message smart object with incoming message
+   * @return value of message param if exists otherwise returns empty string
+   */
+  std::string GetMessageParamValue(const MessageSharedPtr& message) const;
```

Comment:
@AKalinich-Luxoft extremely unclear what is param. Please either rename or add better description